.cirrus.yml

```diff
@@ -101,6 +101,26 @@ fedora_packaging_task:
 
     timeout_in: '20m'
 
+# Verify calls to bin/config were saved
+config_task:
+    env:
+        GOSRC: $CIRRUS_WORKING_DIR
+    # Runs within Cirrus's "community cluster"
+    container:
+        matrix:
+               # fedora:28 doesn't have go mod by default
+               # and we should only need one check to make sure
+               # config changes were synced
+            image: "registry.fedoraproject.org/fedora:29"
```

Comment:
Adding another container image will become a maintenance and debugging burden. I suggest adding any packages you need into `contrib/gate/Dockerfile`.  Then re-use "quay.io/libpod/gate:latest" here, which is automatically built.  The default entrypoint (literally `entrypoint.sh`) runs `make $@` (as in gating task above).

Reply:
Note: This will make PR testing break, so we'll need to merge manually.  There's no automated workflow for testing auto-built images, but as-is we can see it working (until you fix it).  Make sense?

Reply:
"this will make"...meaning, my suggested change.